sdk/src/main/java/io/opentelemetry/sdk/trace/MultiSpanProcessor.java

```diff
@@ -16,15 +16,29 @@
 
 package io.opentelemetry.sdk.trace;
 
+import io.opentelemetry.internal.Utils;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 
-final class MultiSpanProcessor implements SpanProcessor {
+/**
+ * Implementation of the {@code SpanProcessor} that simply forwards all received events to a list of
+ * {@code SpanProcessor}s.
+ */
+public final class MultiSpanProcessor implements SpanProcessor {
```

Comment:
/cc @mayurkale22 this may be what you need.

Reply:
ACK.